torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -76,6 +86,9 @@ def _dequantize_tensor_list(tensor_list, qtype, quant_loss=None):
     elif (qtype == DQuantType.FP16):
         dequantized_tensor_list = [_dequantize_tensor(t, qtype) for t in tensor_list]
         return dequantized_tensor_list
+    elif (qtype == DQuantType.BFP16):
```

Comment:
again, `_dequantize_tensor_list` seems could just call the `dequantize_tensor` directly, we don't need to check the qtype eligibility here.